### DISCUSSION TITLE
Weird setting idea

### DISCUSSION THREAD

**Discussion by Meso22** [niels747/2D-Weather-Sandbox#58] [Ideas]
A land temp setting just like the water temp setting so that the water vapor doesn't cool down the map at night and cover the map in scud that blocks sunlight. Yeah a very unrealistic Idea of mine but a handy tool!

**Comment by niels747** [comment]
Having the temperature of land be fixed is wierd concept, I didn't add realistic solar heating and ir cooling for nothing

**Comment by Meso22** [comment]
Sorry for the stupid question, your sim is very realistic and I love it! I think I made it sound like the land temp was broken (which it's not).

**Reply by Meso22** [reply]
plus the new update is really awesome!

**Reply by niels747** [reply]
It wasn't a stupid question, I just don't think it's a good idea to implement that. I will however make the temperature of each water body  independently settable.

**Reply by Meso22** [reply]
sure that sounds pretty cool!
